ckanext/cnra_schema/schemas/dataset.yaml

```diff
@@ -279,58 +279,135 @@ dataset_fields:
+  #                         but instead of a proper field, it's the name of a Custom Field to be found or created.
+  # extent_field_custom -- Same as extent_field,
+  #                        but instead of a proper field, it's the name of a Custom Field to be found or created.
+  # spatial_coverage_textbox -- A markdown/text field to populate with the names of list of selected places.
+  #                             A button will be created, and clicking this will replace the text field content with a list of selected areas.
+  #                             The spatial_coverage_tagfield_buttontext option specifies the text of the button.
+  # spatial_coverage_tagfield -- A autocomplete/Select2 field to populate with tags listing the selected places.
+  #                              This is similar in ida to the spatial_coverage_textbox, but instead of a text box a tag entry UI is expected.
+  #                              The spatial_coverage_textbox_buttontext option specifies the text of the button.
+  #
+  # As described above, the spatial data may be loaded from several potential fields.
+  # In manage mode this widget will check for some data to pre-populate itself. These fields will be checked, in this order of preference:
+  # - the field specified by "load_geojson_field"
+  # - the four sub-fields of the composite field specified by "load_bbox_composite_field"
+  # - the field specified by "extent_field"
+  # - the field specified by" extent_field_custom"
+  #
+  load_geojson_field: spatial_details
+  geojson_field: spatial_details
+  #extent_field: bboxgeometry
+  #geojson_field_custom: customjson
+  extent_field_custom: spatial
+  #spatial_coverage_textbox: spatial_coverage
+  #spatial_coverage_textbox_buttontext: Copy to Spatial Coverage field
+  spatial_coverage_tagfield: placeKeywords
+  spatial_coverage_tagfield_buttontext: Copy to Place Keywords
+  load_bbox_composite_field: boundingCoordinate
+  load_bbox_composite_south: southBoundingCoordinate
+  load_bbox_composite_north: northBoundingCoordinate
+  load_bbox_composite_east: eastBoundingCoordinate
+  load_bbox_composite_west: westBoundingCoordinate
+  save_bbox_composite: true
 
 ## "spatial_coverage"
 - field_name: spatial_coverage
   label: Spatial Coverage
   form_snippet: markdown.html
   form_placeholder: Name of defined area or geometry of area data describes.
 
-## "spatial_details"
-#- field_name: spatial_details
-#  label: Spatial Coverage Details
-#  form_snippet: text.html
-#  form_placeholder: Filled in automatically as you select areas above.
-#  display_snippet: spatial_details.html
-#  validators: scheming_required scheming_valid_json_object
+## https://github.com/OpenGov-OpenData/ckanext-cnra_schema/pull/6  April 17 2019, the widget should be disabled
+## Enabling these for development of the Gazetteer Widget display, for when it is broght into the website
+## "gazetteer_widget"
+- field_name: gazetteer_widget
```

Comment:
Is the second `gazetteer_widget` suppose to appear here, should the `spatial_details` field be here instead?

Reply:
Sorry I missed this notification when you posted it. Definitely something wrong here. Looking into it . Looks like [this commit](https://github.com/GreenInfo-Network/ckanext-cnra_schema/commit/f6af550adc8e39ce6b4520f34d0d5e204a513308#diff-9b97028e96113dc784ed40f292a2b116) created the error, though odd in that we tested the YML file in our own CKAN environment with the Place Keyword tag display functioning as expected, when I would think this change would have broken our local testing env as it currently is on CNRA Staging (where the widget is trying to load twice and console shows errors about spatial_details). Looking at a fix presently.